contract/src/airdrop/prepareAirdropKit.js

```diff
@@ -0,0 +1,341 @@
+        getAirdropTokenDetails: M.call().returns(AirdropIssuerDetailsShape),
+        claim: M.call(M.string()).returns(M.any()),
+      }),
+      helpers: M.interface('Helper Facet', {
+        getClaimPeriodP: M.call().returns(M.promise()),
+
+        getInternalDepositFacet: M.call().returns(DepositFacetShape),
+        createPurse: M.call().returns(PurseShape),
+        setAirdropWaker: M.call().returns(M.promise()),
+      }),
+    },
+    (claimWindow, hash, claimeeStore, internalPurse) => {
+      return {
+        claimWindowTimeframe: claimWindow,
+        rootHash: hash,
+        claimedUsersStore: claimeeStore,
+        internalPurse,
+        totalTokensClaimed: 0n,
+        distributionSchedule,
+        currentEpoch: distributionSchedule[0],
+      };
+    },
+    {
+      helpers: {
+        createPurse() {
+          return this.state.issuer.makeEmptyPurse();
+        },
+        getInternalDepositFacet() {
+          return this.state.internalPurse.getDepositFacet();
+        },
+        async setAirdropWaker() {
+          await E(contractTimer).setWakeup(
+            privateArgs.claimWindowStartTime,
+            makeWaker('airdropExpirationWaker', () => {
+              // burn tokens from purse
+            }),
+          );
+        },
+        async getClaimPeriodP(p) {
+          return Far('claimPeriodPromise', {
+            getClaimPeriodP: () => p,
+          });
+        },
+      },
+      creator: {
+        makeOpenClaimingWindow() {
+          console.group('---------- inside makeOpenClaimingWindow----------');
+          console.log('------------------------');
+          console.log('this.state::', this.state);
+          console.log('------------------------');
+          console.log('::');
+          console.log('------------------------');
+          console.groupEnd();
+
+          /** @type {OfferHandler} */
+          const startAirdropHandler = async (seat, offerArgs) => {
+            const {
+              give: { Deposit },
+            } = seat.getProposal();
+            atomicRearrange(zcf, harden([[seat, contractSeat, { Deposit }]]));
+
+            console.log(
+              'inside startAirdropHandler',
+              contractSeat.getCurrentAllocation(),
+            );
+
+            seat.exit();
+            void this.facets.helpers.setAirdropWaker();
+            return 'successfully opened claiming window';
+          };
+
+          return zcf.makeInvitation(startAirdropHandler, 'startAirdropHandler');
+        },
+        getPurse() {
+          return airdropPurse;
+        },
+        async depositToPurse(payment) {
+          const depositFacet = await E(airdropPurse).getDepositFacet();
+          await E(depositFacet).receive(payment);
+          return 'Deposit success!';
+        },
+        addEligibleUsers(list) {
+          const { claimedUsersStore: store } = this.state;
+          store.addAll(list);
+          return AIRDROP_ADMIN_MESSAGES.ADD_ACCOUNTS_SUCCESS(list);
```

Comment:
a function named with ALL CAPS? odd.

Reply:
I agree an all caps function is odd.

iirc, this value evolved from being a static string (which are values I tend to declare with all caps variable names)